Feature/gatk4 sif (#1099)
Clinical-Genomics/MIP: README.md

This will generate a bash script called "mip.sh" in your working directory.

###### *Note:*

>   The batch script will attempt to install the MIP dependencies in a conda environment called MIP_rare. MIP will install the following environments by default:

Reviewer: ```suggestion
  By default the batch script will attempt to install the MIP dependencies in a conda environment called mip7_rd-dna.
```

>   * mip7_rd-dna

Reviewer: ```suggestion
  * mip7_rd-dna
```
```suggestion

```

> It is possible to specify which environments to install using the ``--installations`` flag, as well as the names of the environments using the ``--environment_name`` flag. E.g. ``--installations emip --environment_name emip=MIP``.   

Reviewer: ```suggestion
It is possible to specify the name of the environment using the ``--environment_name`` flag. E.g. --environment_name emip=MIP``.
```
